docs/tutorials/vis_cpu_tutorial.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Also showing a plot as a function of time/LST might be nice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/vis_cpu/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
I have modified the notebook as suggested.